packages/terra-brand-footer/src/terra-dev-site/test/brand-footer/HorizontalWrapBrandFooter.test.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import classNames from 'classnames/bind';
 
 import BrandFooter from '../../../BrandFooter';
+import styles from './HorizontalWrapBrandFooter.test.scss';
+
```

Comment:
import missing for BrandFooterGTestCommon

Reply:
This is intentional, "HorizontalWrapBrandFooter.test.scss" already imports "BrandFooterTestCommon.scss".
The reason for having a separate scss file for this was because "HorizontalWrapBrandFooter.test.jsx" has an extra element with a  unique style which was not present in the other files.

Reply:
I think it is okay to reuse the same style sheet for multiple examples even if all styles are not used for a particular example. Overall it reduces the amount of code to maintain

Reply:
Removed "HorizontalWrapBrandFooter.test.scss" and using only "BrandFooterTestCommon.scss".